prusti-viper/src/encoder/type_encoder.rs

```diff
@@ -758,6 +758,7 @@ pub fn compute_discriminant_values<'tcx>(adt_def: &'tcx ty::AdtDef, tcx: ty::TyC
     discr_values
 }
 
+
```

Comment:
There are many of these whitespace-only changes in this PR, maybe could be removed?